docs/setup/extension.md

```diff
@@ -0,0 +1,72 @@
+Extension de la plataforma `DatosGobAr`:
+
+Plugins
+---
+
+La plataforma `CKAN` al ser instalada, por omisión, almacena todo su código fuente en la `PATH=/usr/lib/ckan/default/src`, homologamente, Andino X y DatosGobAr funcionan de la misma manera, por tanto dentro de su contenedor principal `datosgobar` y `andino` respectivamente, será dispuesto todo su código fuente.
+
+Así mismo, las extensiones son guardadas con un formato particular, predefinido por el Framework utilizado para el desarrollo de CKAN(Pylons), siendo las mismas  almacenadas bajo la siguiente regla:
+
+  /usr/lib/ckan/default/src/ckanext-{nombre_de_la_extension}/ckanext/{nombre_de_la_extension}
+
+_Y responden al árbol:_
+
+```
+  ckanext-{nombre_de_la_extension}/
+  ├── setup.py
+  ├── pip-requirements.txt
+  ├── README.md
+  ├── ckanext
+  │   └── {nombre_de_la_extension}
+  │       ├── ...
+  │       ├── plugin.py
+  |       ├── Controller{Nombre_de_la_extension}.py
+  │       └── tests
+  └── ckanext_{nombre_de_la_extension}.egg-info
+```
+### Cambiar la Branch.
+
+```bash
```

Comment:
En vez de poner el código todo junto en bash, explicaría cuáles son los pasos que hay que hacer. 
Ejemplo:
1. Ingresar al contenedor correspondiente
`docker exec -it andino /bin/bash` para andino
`docker exec -it datosgobar /bin/bash` para datos.gob.ar

2. Ir al directorio de la extensión que se quiere testear
`cd /usr/lib/ckan/default/src/ckanext-{nombre_de_la_extension}`

3. Hacer checkout de la branch a partir de la cual se quieren testear cambios
```
git fetch
git checkout {branch-en-cuestion}
```

4. Reiniciar servidor de Apache
`service apache2 reload`